spec/datadog/tracing/transport/trace_formatter_spec.rb

```diff
@@ -237,6 +237,23 @@
         end
       end
 
+      shared_examples 'first span with process tags' do
+        it do
+          format!
+          expect(first_span.meta).to include('_dd.tags.process')
+          expect(first_span.meta['_dd.tags.process']).to eq(Datadog::Core::Environment::Process.serialized)
+          # TODO figure out if we need an assertion for the value, ie
```

Comment:
@marcotc - do you think there's value in asserting for the values of the tag? Or is the test in process_spec enough?

Reply:
What you are doing with `          expect(first_span.meta['_dd.tags.process']).to eq(Datadog::Core::Environment::Process.serialized)` seems good to me.

Reply:
Thanks! In that case it doesn't seem like I need to make any changes to the assertions then?